resultsdbupdater/consumer.py

```diff
@@ -22,28 +22,29 @@ def log_msg(self, msg):
     def consume(self, msg):
         # Some of the messages here can be empty strings, so only process
         # them if they are dicts to avoid tracebacks
-        if not isinstance(msg['body']['msg'], dict):
+        msg_body = msg['body']['msg'] if 'body' in msg else msg.get('msg', {})
```

Comment:
Ah I see. You probably have this so that your tests pass. So fedmsg-hub actually reformats the message format into the following:
```
{
  "topic": "...",
  "headers": {...},
  "body": {
    "headers": {...},
    "msg": {actual message content}
  }
}
```

Reply:
Yes, so fedmsg-hub reformats the message as described above.